modules/javafx.graphics/src/test/java/test/com/sun/javafx/test/CssMethodsTestBase.java

```diff
@@ -109,8 +110,8 @@ public static Object[] config(
                                         comparator));
     }
 
-    public static Object[] config(final Configuration configuration) {
-        return new Object[] { configuration };
+    public static Arguments config(final Configuration configuration) {
```

Comment:
I think this can be simplified: use Configuration directly, no need to wrap it into Arguments.
But this code will work too.

Reply:
Changed it to `Configuration` and all parameter method sources to `Stream<Configuration>`